python/pyspark/worker.py

```diff
@@ -837,6 +837,26 @@ def wrap_window_agg_pandas_udf(
         )
 
 
+def wrap_window_agg_arrow_udf(f, args_offsets, kwargs_offsets, return_type, runner_conf, udf_index):
+    window_bound_types_str = runner_conf.get("pandas_window_bound_types")
```

Comment:
`pandas_window_bound_types`? Maybe we should have `arrow_window_bound_types`?

Reply:
Arrow UDF reuses the same https://github.com/apache/spark/blob/master/sql/core/src/main/scala/org/apache/spark/sql/execution/python/ArrowWindowPythonEvaluatorFactory.scala#L71

Maybe we can give it a better name